TYPO3.Flow/Classes/TYPO3/Flow/Configuration/ConfigurationManager.php

```diff
@@ -125,11 +125,11 @@ class ConfigurationManager
      * @var array
      */
     protected $configurationTypes = [
+        self::CONFIGURATION_TYPE_SETTINGS => ['processingType' => self::CONFIGURATION_PROCESSING_TYPE_SETTINGS, 'allowSplitSource' => false],
```

Comment:
Does changing the order of types have an impact of the compiled configuration in general? Or did you change the order for other reasons?